lms/djangoapps/grades/new/course_grade.py

```diff
@@ -281,13 +349,20 @@ def _get_saved_grade(self, course, course_structure):  # pylint: disable=unused-
         """
         Returns the saved grade for the given course and student.
         """
-        if PersistentGradesEnabledFlag.feature_enabled(course.id):
-            # TODO LATER Retrieve the saved grade for the course, if it exists.
-            _pretend_to_save_course_grades()
+        if not PersistentGradesEnabledFlag.feature_enabled(course.id):
+            return None
 
+        saved_course_grade = CourseGrade.load_persisted_grade(
+            self.student,
+            course,
+            course_structure
+        )
+        return saved_course_grade
```

Comment:
Nitty (feel free to ignore): Can `return` on the above line instead of creating a temp variable.